Formula/stubby.rb

```diff
@@ -0,0 +1,84 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver service based on getdns"
+  homepage "https://getdnsapi.net/blog/dns-privacy-daemon-stubby/"
+  url "https://getdnsapi.net/releases/getdns-1-1-1/getdns-1.1.1.tar.gz"
+  sha256 "fa414c30d5f2d2b2453b5cec77362b4cc0f44d440be5893233748d82bd6a1a56"
+
+  head do
+    url "https://github.com/getdnsapi/getdns.git", :branch => "develop"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on "openssl"
+  depends_on "libevent" => :recommended
+  depends_on "libev" => :optional
+  depends_on "libuv" => :optional
+
+  def install
+    # Current Makefile layout prevents simultaneous job execution
+    # https://github.com/getdnsapi/getdns/issues/166
+    ENV.deparallelize
+
+    if build.head?
+      system "glibtoolize", "-ci"
+      system "autoreconf", "-fiv"
+    end
+
+    args = [
+      "--with-ssl=#{Formula["openssl"].opt_prefix}",
+      "--with-trust-anchor=#{etc}/getdns-root.key",
+    ]
+    args << "--with-libevent" if build.with? "libevent"
+    args << "--with-libev" if build.with? "libev"
+    args << "--with-libuv" if build.with? "libuv"
+
+    system "./configure", "--prefix=#{prefix}",
+                          "--enable-stub-only",
+                          "--without-libidn",
+                          "--enable-debug-daemon",
+                          *args
+    system "make", "install"
+
+    etc.install "src/tools/stubby.conf"
+    bin.install "src/tools/stubby-setdns-macos.sh"
+  end
+
+  plist_options :startup => true, :manual => "sudo stubby -C #{HOMEBREW_PREFIX}/etc/stubby.conf"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-/Apple/DTD PLIST 1.0/EN" "http:/www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>KeepAlive</key>
+        <true/>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_bin}/stubby</string>
+          <string>-C</string>
+          <string>#{etc}/stubby.conf</string>
+        </array>
+        <key>UserName</key>
+        <string>root</string>
+        <key>StandardErrorPath</key>
+        <string>#{var}/log/stubby/stubby.log</string>
+        <key>StandardOutPath</key>
+        <string>#{var}/log/stubby/stubby.log</string>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    (testpath/"test_message.txt").write("getdnsapi.net")
+    output = shell_output("#{bin}/stubby -C #{etc}/stubby.conf -z 127.0.0.1:5553 -e 0 -F test_message.txt 2>/dev/null")
```

Comment:
Unrelated to why it's getting stuck, but this should probably be 2>&1 so both stdout and stderr get printed when there's an assertion failure